ukol2/ukol2.py

```diff
@@ -1,67 +1,71 @@
 import requests
 
 
-def vyhledat_podnikatelsky_subjekt_ico():
-    ico = input("Zadejte ICO: ")
+def search_subject(ico: str) -> str:
```

Comment:
i přes to, že se nebrobíralo, je dobré uvádět, co funkce vrací tedy " -> str" napovídá, že bude vracet string - obecně by funkce vždy měla něco vracet